packages/app/src/cli/services/dev/extension/server/utilities.ts

```diff
@@ -60,11 +60,10 @@ function getCustomerAccountsRedirectUrl(
   options: ExtensionDevOptions,
   requestedTarget = '',
 ): URL {
-  const [storeName, ...storeDomainParts] = options.storeFqdn.split('.')
```

Comment:
Just curious, was the storeFqdn being used for anything else beyond parsing out it components to construct the old customer account domain? If not, can we remove it?